src/app/components/actions/actions.tsx

```diff
@@ -1,15 +1,26 @@
+import { TextWithLink } from '../text-with-link';
+import { Description } from '@/features/required/required.types';
+
 import './actions.scss';
 
 interface ActionsProps {
-  actions: string[];
+  actions: Description;
   marked?: boolean;
 }
+
 export const Actions = ({ actions, marked = false }: ActionsProps) => {
   return (
-    <ul className={`stage-actions ${marked ? 'marked' : ''}`}>
-      {actions.map((action, index) => (
-        <li key={index}>{action}</li>
-      ))}
+    <ul className="stage-actions">
+      {actions.map((action, index) => {
+        const isLink = typeof action !== 'string';
```

Comment:
I don't see any reason to declare variables in a returne

Reply:
Please consider to give an example on how to save the code functionality and readability without a constants here